client/my-sites/site-settings/disconnect-site/confirm.jsx

```diff
@@ -61,7 +62,7 @@ class ConfirmDisconnection extends PureComponent {
 	render() {
 		const { reason, siteId, siteSlug, translate } = this.props;
 		const previousStep = find(
-			without( keys( this.constructor.reasonWhitelist ), 'troubleshooting' ), // There's no step for Troubleshooting
+			without( keys( this.constructor.reasonWhitelist ), 'troubleshooting', 'other' ), // Redirect those back to initial survey
```

Comment:
I find this `without keys` to be a bit hard to understand. Do you think the logic would be clearer in the `find` predicate? I think the following is synonymous:

```js
find(
  this.constructor.reasonWhitelist,
  r => ! includes( [ 'troubleshooting', 'other' ], r ) && r === reason,
)
```

Reply:
I kinda prefer Bernie's version to this one 😉  Seems more readable to me.

Reply:
👍  fair enough, retracted 🙂